src/utils/runtime/src/lib.rs

```diff
@@ -199,9 +216,7 @@ pub fn init_risingwave_logger(settings: LoggerSettings, registry: prometheus::Re
         layers.push(
             fmt_layer
                 .with_filter(FilterFn::new(|metadata| metadata.is_event())) // filter-out all span-related info
-                .with_filter(filter(vec![
-                    ("rw_tracing", Level::OFF), // filter out tracing-only events
-                ]))
+                .with_filter(default_filter().with_target("rw_tracing", Level::OFF)) // filter-out tracing-only events
```

Comment:
Would it be better to opt-in rw_tracing in tracing layer instead of opting-out here?

Reply:
Do you mean setting `rw_tracing` to `OFF` in `default_filter`?

Reply:
yes

Reply:
I don't find a way to "cancel" an `OFF` elegantly, or we have to retrieve the default level and set it again. 😕